MythicMobs/Mobs/pinkerium

```diff
@@ -31,3 +31,37 @@ pinkubus:
   LevelModifiers:
   Disguise:
   Skills:
+
+syncophant: 
+  Type: zombie
+  Display: '&d Pink Syncophant&r - &k<mob.level>'
+  Health: 300
+  Damage: 4
+  Armor: 3
+  BossBar:
```

Comment:
Is it okay for you to leave fields empty like this? I assume it would be, though I just wanna be sure.

Reply:
Also, by leaving bossbar blank, would this mean that the syncophant would display its bossbar whenever someone is nearby? I feel that your boss mobs would feel extra special if the bossbar was used sparingly. That being said, if the syncophant is intended to be a boss, and not a swarm mob, (I notice health:300), then a bossbar would be quite helpful.

Reply:
Anything I end up not using will be commented before deployment. The only boss would be Pepperglitter, with the enforcers as potentially minibosses.